hummingbird/ml/operator_converters/decision_tree.py

```diff
@@ -204,10 +204,28 @@ def convert_sklearn_decision_tree_classifier(operator, device, extra_config):
     operator.raw_operator.estimators_ = [operator.raw_operator]
     return convert_sklearn_random_forest_classifier(operator, device, extra_config)
 
+def convert_sklearn_decision_tree_regressor(operator, device, extra_config):
+    """
+    Converter for `sklearn.tree.DecisionTreeRegressor`.
+
+    Args:
+        operator: An operator wrapping a `sklearn.tree.DecisionTreeRegressor` model
+        device: String defining the type of device the converted operator should be run on
+        extra_config: Extra configuration used to select the best conversion strategy
+
+    Returns:
+        A PyTorch model
+    """
+    assert operator is not None
+
+    operator.raw_operator.estimators_ = [operator.raw_operator]
+    return convert_sklearn_random_forest_regressor(operator, device, extra_config)
+
 
 # Register the converters.
 register_converter("SklearnRandomForestClassifier", convert_sklearn_random_forest_classifier)
 register_converter("SklearnRandomForestRegressor", convert_sklearn_random_forest_regressor)
 register_converter("SklearnDecisionTreeClassifier", convert_sklearn_decision_tree_classifier)
+register_converter("SklearnDecisionTreeRegressor", convert_sklearn_decision_tree_regressor)
 register_converter("SklearnExtraTreesClassifier", convert_sklearn_random_forest_classifier)
 register_converter("SklearnExtraTreesRegressor", convert_sklearn_random_forest_regressor)
```

Comment:
(Aside) We should probably have its own helper method for maintainability and readability.

Reply:
You mean for decision trees-based models? What do you have in mind?

Reply:
Extra tree (classifier/regressor) call the helper methods of random forest XXXX directly.
I was of opinion that we should create helper methods:
`convert_sklearn_extra_trees_classifier` and `convert_sklearn_extra_trees_classifier`
Although there is some duplication, it is easier to read and is more customisable (if we need any customisation in future)